.github/workflows/sheduled_cleanup.yaml

```
@@ -47,3 +47,17 @@ jobs:
         if: always()
         run: |
           rm -f /tmp/hub-tool-linux-amd64.tar.gz /tmp/hub-tool /tmp/auth.json ~/.docker/config.json
+
+  docker-installation_jfrog_cli:
```

Can you change this name? To something like 
```suggestion
  jfrog_packages_cleanup:
```